.github/workflows/build_and_test.yaml

```diff
@@ -47,3 +47,33 @@ jobs:
       run: cd build/debian/cuttlefish_cvdremote && dpkg-buildpackage -i -uc -us -b
     - name: Install package
       run: dpkg -i build/debian/cuttlefish-cvdremote_*_*64.deb || apt-get install -f -y
+
+  build-cuttlefish-cloud-orchestration-x86_64-docker-image:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675 # aka v2
+    - name: Build docker image
+      run: bash scripts/docker/image-builder.sh
+    - name: Save docker image
+      run: docker save --output cloud-orchestration-docker-image-x86_64.tar cuttlefish-cloud-orchestration:latest
+    - name: Publish docker image
+      uses: actions/upload-artifact@c7d193f32edcb7bfad88892161225aeda64e9392 # aka v4.0.0
+      with:
+        name: cloud-orchestration-docker-image-x86_64
+        path: cloud-orchestration-docker-image-x86_64.tar
+
+  build-cuttlefish-cloud-orchestration-arm64-docker-image:
```

Comment:
let's remove this artifact for now until needed. There's an option where we don't need the CO in the arm fleet. Let me finish the details first.

Reply:
It's needed for both x86_64 and arm64. CO should be run on the same machine running docker instances with HOs. Running CO in distinguished server would be the option in the future with a small fix I think, but it's not available right now.

Reply:
Docker image would be built for both x86_64 and arm64 archs. It would be managed by manifest registered in Artifact Registry like orchestration image did. Resolving outdated comment.